### ISSUE TITLE
SIGSEGV in Lab

### ISSUE THREAD

**Issue by ejseto** [CleverRaven/Cataclysm-DDA#10791]
Was minding my own business, crafting in a lab, when I got a SIGSEGV crash. Appears to be due to the monster uncanny dodge changes. Seems fairly reproducible if you just spend some time in a lab letting the monsters fight each other.

```
#0 007C8CF1	mattack::science(this=0xdc8f92f, z=0x1c9e1b10, index=0) (C:\Cata\src\monattack.cpp:486)
#1 007F024B	monster::move(this=0x1c9e1b10) (C:\Cata\src\monmove.cpp:268)
#2 005088CA	game::monmove(this=0x18a63018) (C:\Cata\src\game.cpp:6637)
#3 004E626F	game::do_turn(this=0x18a63018) (C:\Cata\src\game.cpp:1484)
#4 0066D430	main(argc=0, argv=0x25180c) (C:\Cata\src\main.cpp:296)
```

**Comment by narc0tiq** [comment]
Off-topic, but I love that scientist zombies attack you with !!SCIENCE!!, which is not something I'd noticed before. That is all.

**Comment by bkentel** [comment]
Was this resolved by 0a88fbb6d244958a26b731cdd9bb09ccece482e5 ?

**Comment by Coolthulhu** [comment]
The `mattack::science` function was rewritten some time after this bug was submitted, so I'm pretty sure it's safe to close this issue.
